package/src/components/TrackMark.tsx

```diff
@@ -4,22 +4,27 @@ import {styles} from '../utils/styles';
 
 export type MarkerProps = {
   stepMarked: boolean;
+  currentValue?: number;
 };
 
 export type TrackMarksProps = {
   isTrue: boolean;
   thumbImage?: ImageURISource;
   StepMarker?: FC<MarkerProps>;
+  currentValue?: number;
 };
 
 export const SliderTrackMark = ({
   isTrue,
   thumbImage,
   StepMarker,
+  currentValue,
```

Comment:
Yes, I was thinking that at some point the current value should be added, so that it's possible to know where the thumb is from the level of StepMarker. I'm glad it also helps to fix this issue.